### PR TITLE
Fixed DE symbols & Revisited GridSet 

### PR DESCRIPTION
**Pull request by quaxalber**

### CODE REVIEW

**Comment by quaxalber** [comment]
@willwade Curious for your feedback. Would adapt the GridSet for US and UK accordingly

File: cli_keymaps/de_keymap.json
Comment (quaxalber) [inline cli_keymaps/de_keymap.json:1]: The `ALTGR` modifier is not defined in [blehid.py](https://github.com/AceCentre/RelayKeys/blob/master/blehid.py). `RALT` is the correct key name.

**Comment by willwade** [comment]
This looks good @quaxalber - love the DE GridSet. Are you finished for now? If you want/can (!) it would be great to Migrate the English one to the same layout but I can merge for now and do that later if needed

**Comment by quaxalber** [comment]
Glad you like it. I'm quite happy with the result myself. Didn't expect the pieces come together so nicely. I will continue with EN but from my side you could merge already.

**Pull request merged**